### DISCUSSION TITLE
How to make sesh window transparent?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [joshmedeski/sesh#200] [Q&A] [answered]
I'm using zsh+kitty+tmux+fzf. The sesh floating window integrated with fzf appears as expected ,but it's not transparent. Is it possible to make it transparent?

**Comment by joshmedeski** [comment, marked as answer]
Yes, you can override the color's `bg` value to `-1` and it will make it transparent.
(I'm on fzf version 0.56.3 (brew))

See [my FZF_DEFAULT_OPTS]( 
https://github.com/joshmedeski/dotfiles/blob/9bcd9e8588d0c8f75bf3374a8bc272dfb55414bf/.config/fish/conf.d/fzf-custom.fish?plain=1#L15)

**Reply by wojciech-kulik** [reply]
Thank you! Exactly what I needed 🍻  Thank you for the great plugin!
